engine/src/main/resources/templates/ResourceOptimization/dailyEmailBody.html

```diff
@@ -0,0 +1,46 @@
+{#include ResourceOptimization/insightsEmailBody}
+{#content-title}Daily Resource Optimization Service Summary - {action.context.start_time.toStringFormat()}{/content-title}
```

Comment:
I suppose this is needed too.
```suggestion
{#content-title}Daily Resource Optimization Summary - {action.context.start_time.toStringFormat()}{/content-title}
```

Reply:
Thanks for pointing!